public/docs/ts/latest/guide/i18n.jade

```diff
@@ -0,0 +1,302 @@
+include ../_util-fns
+
+:marked
+  With internationalization, also known as i18n, we can display our website in multiple languages. There are two different
```

Comment:
There is nothing written in stone. At the end the lines gets wrapped in the HTML